Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -225,9 +237,14 @@ private async Task<IEnumerable<UserProfile>> GetDelegateProfilesAsync(string dep
             if (hdid != null)
             {
                 profile = await this.userProfileDelegate.GetUserProfileAsync(hdid).ConfigureAwait(true);
+
+                if (patient == null && profile == null)
+                {
+                    return null;
+                }
             }
 
-            if (patient == null && profile == null)
+            if (patient == null)
             {
                 return null;
             }
```

Comment:
The `patient == null` if statement can be moved after the call to retrieve the patient.  Then the other if statement can be simplified from `patient == null && profile == null` to just `profile == null`.